src/policy/copyspace.rs

```diff
@@ -84,7 +94,7 @@ impl<VM: VMBinding> CopySpace<VM> {
                 vmrequest,
                 side_metadata_specs: SideMetadataContext {
                     global: global_side_metadata_specs,
-                    local: vec![],
+                    local: local_specs,
```

Comment:
For cases like this, you can have a function `fn extract_side_metadata(specs: &[MetadataSpec]) -> Vec<SideMetadataSpec>`.

Reply:
Probably check the constructors for all the policies.

Reply:
Done